web/packages/teleterm/src/services/vnet/testHelpers.ts

```diff
@@ -56,7 +57,20 @@ export const makeCheckReport = (
 ): CheckReport => ({
   status: CheckReportStatus.OK,
   report: {
-    oneofKind: undefined,
+    oneofKind: 'routeConflictReport',
```

Comment:
is oneof supposed to be like this or `oneOf` ?

Reply:
It's supposed to be like this, it's generated by protobuf-ts. In protos it's also called `oneof` and not `oneOf`. https://protobuf.dev/programming-guides/proto3/#oneof